include/aca_vlan_manager.h

```diff
@@ -41,7 +41,7 @@ struct vpc_table_entry {
   // hashtable <key: outports string, value: list of neighbor port IDs>
   unordered_map<string, list<string> > outports_neighbors_table;
 
-  uint oam_server_port;
+  string auxGateway_id;
```

Comment:
we will think about how to "encapsulate" zeta gateway info into this data structure, because we will have other aux gateways in the future.